contributed_definitions/nyaml/NXscan_control.yaml

```diff
@@ -0,0 +1,63 @@
+category: base
+doc: |
+  The scan_control group contains information about the scan control over a certain area.
+scan_control:
+  exists: optional
+  scan_range(NX_NUMBER):
```

Comment:
as discussed, first the independent phisical dimensions shall be defined, than th scna type and then the parameters for the different type of scnas.